newIDE/app/src/EventsBasedBehaviorEditor/EventsBasedBehaviorPropertiesEditor.js

```diff
@@ -267,6 +272,15 @@ export default class EventsBasedBehaviorPropertiesEditor extends React.Component
                                 />
                               </SelectField>
                             )}
+                            {property.getType() === 'Choice' && (
+                              <StringSelectorEditor
+                                extraInfo={property.getExtraInfo()}
+                                setExtraInfo={newExtraInfo => {
```

Comment:
Ah good catch, I did not see that StringSelectorEditor was made for parameters, which store their additional information into a string, using `setExtraInfo`. Properties have the option to store multiple extra info in an array (a "vector" in C++). So we must adapt this. I would suggest to rework StringSelectorEditor to work with a JS array, and have the code calling it to handle the parsing from/to a JSON array (for parameters) or from/to `getExtraInfo`/`addExtraInfo` (for properties).

@ClementPasteau will look at this I think! :)